config/plugin/windows_defender_problem.ps1

```diff
@@ -0,0 +1,15 @@
+# This plugin checks to see if windows defender detects any threats to the node.
+
+$windowsDefenderThreats = Get-MpThreat
+$currentThreatDetected = $false
+
+foreach ($threat in $windowsDefenderThreats) {
```

Comment:
You can condense this to 
`$windowsDefenderThreats = (Get-MpThreat | Where-Object {$_.IsActive -or $_.DidThreatExecute})`